_data/simple-icons.json

```
@@ -6588,6 +6588,11 @@
             "source": "https://mozilla.design/mozilla/",
             "guidelines": "https://mozilla.design/mozilla/"
         },
+        {
+            "title": "mpv",
+            "hex": "520053",
+            "source": "https://commons.wikimedia.org/wiki/File:Mpv_logo_(official).png"
```

```suggestion
            "source": "https://github.com/mpv-player/mpv/blob/da400ed3a9b54408d1b5112855a7281380a1ef52/etc/mpv.svg"
```
SVG from GitHub repo